src/style.css

```
@@ -0,0 +1,29 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #000;
+}
+
+.outsideCollor {
```

don't use camelcase, you kebab case
```suggestion
.outside-color {
```

why have u added `color` word?